toolkit/tools/internal/safemount/safemount.go

```diff
@@ -16,9 +17,9 @@ import (
 )
 
 type Mount struct {
-	target     string
-	isMounted  bool
-	dirCreated bool
+	target      string
+	isMounted   bool
+	fileCreated bool
```

Comment:
nit: sounds like this is `fileOrDirCreated`?

Reply:
Technically directories are a type of file. But `fileOrDirCreated` is probably clearer.

Reply:
Fixed